chat-plugins/wisp.js

```diff
@@ -1095,6 +1095,50 @@ exports.commands = {
 		"/background set [user], [image] - Sets a users profile background.",
 		"/background delete [user] - Deletes a users profile background.",
 	],
+	music: {
+		set: function (target, room, user) {
+			if (!this.can('music')) return false;
```

Comment:
you mean a music: true, property to usergroups with access?  Or an actual export?